dataclass_factory/serializers.py

```diff
@@ -26,37 +25,37 @@ def serialize(data):
     return serialize
 
 
-def get_collection_serializer(serializer) -> Serializer:
+def get_collection_serializer(serializer) -> Serializer[List]:
     return lambda data: [serializer(x) for x in data]
 
 
-def get_tuple_serializer(serializers) -> Serializer:
+def get_tuple_serializer(serializers) -> Serializer[List]:
     return lambda data: [serializer(x) for x, serializer in zip(data, serializers)]
 
 
-def get_collection_any_serializer() -> Serializer:
+def get_collection_any_serializer() -> Serializer[List]:
     return lambda data: [x for x in data]
 
 
-def stub_serializer(data):
+def stub_serializer(data: T) -> T:
     return data
 
 
-def get_dict_serializer(serializer):
+def get_dict_serializer(serializer: Any) -> Serializer[Dict]:
     return lambda data: {
         k: serializer(v) for k, v in data.items()
     }
 
 
-def lazy_serializer(factory):
+def lazy_serializer(factory) -> Serializer:
```

Comment:
Is it possible to type `Serializer`?

Reply:
No. In this case created serializer can serialize any data, beacuse it find concrete serializer in every call